conans/client/command.py

```diff
@@ -889,13 +889,17 @@ def upload(self, *args):
         parser.add_argument('--retry-wait', default=5, type=int,
                             help='Waits specified seconds before retry again',
                             action=OnceArgument)
+        parser.add_argument("-no", "--no-overwrite", type=str, choices=["all", "recipe"],
```

Comment:
I still prefer --no-overwrite without parameters mean = all :)

Lets wait for @lasote feedback on this